ui/src/api/beamline.js

```diff
@@ -14,16 +14,22 @@ export function sendPrepareBeamlineForNewSample() {
   return endpoint.put(undefined, '/prepare_beamline').res();
 }
 
-export function sendExecuteCommand(obj, name, args) {
-  return endpoint.post(args, `/${obj}/command/${name}`).res();
+export function sendExecuteCommand(object_type, object_id, command, args) {
+  return api
+    .url('/hwobj')
```

Comment:
I would make a new API file called `hardware-object.js` and follow the same pattern as in the other files (i.e. `endpoint` variable to initialise a wretch instance with the endpoint's URL)